### DISCUSSION TITLE
Bold lock without Bold connect

### DISCUSSION THREAD

**Discussion by SuperMPtje** [community/community#42158] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, i’m curious to know whether you are considering to create an implementation for Bold without Bold connect? So using a direct bluetooth connection. I saw that something like that is available for Homey.

**Comment by Akash1134** [comment]
Hi @SuperMPtje  ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
